src/entities/removeLiquidity/helper.ts

```diff
@@ -67,7 +64,11 @@ export const getAmountsCall = (
             };
         case RemoveLiquidityKind.SingleTokenExactIn:
             if (input.tokenOutIndex === undefined) {
-                throw removeLiquiditySingleTokenExactInShouldHaveTokenOutIndexError;
+                throw missingParameterError(
+                    'Remove Liquidity SingleTokenExactIn',
+                    'tokenOutIndex',
+                    input.protocolVersion,
+                );
```

Comment:
Ya I'm liking this a lot more than the sentence long var names :clap:

Reply:
I had the same feeling - getting rid of those long variables felt good 😅